README.md

```diff
@@ -33,6 +33,28 @@ Things you might want to know when using this addon:
   * When googletag.pubads().refresh() is called to request a new ad and returns empty, the default GPT behavior is to keep the current ad. Set this to true if you want to `.clear()` the ad slot when empty. 
   * You might want to use this if you're using key/value targeting, and never want a targeted ad to stick around (due to lack of other ads) after you change the targeting.
 
+## Configuration
+
+You can configure the following settings in your app's `environment.js`.
+```js
+module.exports = function(environment) {
+  let ENV = {
+    //...
+    nyprAds: {
```

Comment:
I've seen other addons use the `'dasherized-name'` as the toplevel config key, e.g. `ember-cli-babel`. What do you think?

Reply:
I was trying to find examples earlier and coming up blank, but if hyphens are good enough for `ember-cli-babel` and `ember-cli-mirage` they're good enough for me.